### PR TITLE
165856948 Create Profile Components

### PR DESCRIPTION
**Pull request by chikeozulumba**
#### Description
Create additional components for the design of the UI for the Profile Page.
#### Type of change
- [ ] Bug Fix (a non-breaking change which fixes an issue)
- [x] New Feature (a non-breaking change that adds functionality)
- [ ] New Chore (a non-breaking change that adds functionality)
- [ ] Breaking Change (a fix or feature which would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update
#### How Has This Been Tested?
- [ ] End to End
- [ ] Integration
#### Checklist:
- [ ] Unit test
- [ ] Code coverage
#### PT
[#165856948](https://www.pivotaltracker.com/story/show/165856948)

### CODE REVIEW

File: src/components/ProfileComponent/Social/index.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  margin: 2rem 0;
+`;
+
+const Social = ({ socialLinks }) => (
+	<Fragment>
+		<Wrapper>
+			<Social.Wrapper>
+				<Followers paragraphStyle={{ margin: '0 10px', fontFamily: 'Circular-Book', color: '#3A8FDD' }} followers="90"/>
+				<Following paragraphStyle={{ margin: '0 10px', fontFamily: 'Circular-Book', color: '#3A8FDD' }} following="190"/>
+			</Social.Wrapper>
```
Comment (Temmyogunbo) [inline src/components/ProfileComponent/Social/index.jsx]: Fix indentation problem, @chikeozulumba

File: src/components/ProfileComponent/Social/Icons/index.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	<Fragment>
+		<SocialIcon.Container onClick={handleSocialClick} size={size} src={source}/>
+	</Fragment>
+);
+
+
+SocialIcon.Container = styled.img`
+  width: 20px;
+	height: 16px;
+	cursor: pointer;
```
Comment (Temmyogunbo) [inline src/components/ProfileComponent/Social/Icons/index.jsx]: Fix indentation problem

**Review by Temmyogunbo — CHANGES_REQUESTED** [review]

File: src/components/ProfileComponent/Social/index.jsx (hunk 1 above)
Comment (chikeozulumba) [reply src/components/ProfileComponent/Social/index.jsx]: All fixed @Temmyogunbo

File: src/components/ProfileComponent/Social/Stats/Following.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React, { Fragment } from 'react';
+import { Paragraph } from '../../../shared/Text/Text';
+
+const Following = ({ following, paragraphStyle }) => (
+  <Fragment>
+    <Paragraph paragraphStyle={paragraphStyle}>{following || 10} following</Paragraph>
+	</Fragment>
```
Comment (Temmyogunbo) [inline src/components/ProfileComponent/Social/Stats/Following.jsx]: @chikeozulumba fix indentation

File: src/components/ProfileComponent/Social/Stats/Followers.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React, { Fragment } from 'react';
+import { Paragraph } from '../../../shared/Text/Text';
+
+const Followers = ({ followers, paragraphStyle }) => (
+  <Fragment>
+    <Paragraph paragraphStyle={paragraphStyle}>{followers || 0} followers</Paragraph>
+	</Fragment>
```
Comment (Temmyogunbo) [inline src/components/ProfileComponent/Social/Stats/Followers.jsx]: Fix this too

**Review by Temmyogunbo — CHANGES_REQUESTED** [review]

**Review by Temmyogunbo — APPROVED** [review]

**Pull request merged**